src/fermilib/utils/_sparse_tools_test.py

```diff
@@ -65,12 +66,8 @@ def test_jw_restrict_operator(self):
         n_qubits = 6
         target_electrons = 3
         penalty_const = 100.
-        number_operator = sum([FermionOperator(((i, 1), (i, 0)), 1.0) for i
-                              in range(n_qubits)], FermionOperator())
-        number_sparse = jordan_wigner_sparse(number_operator)
-        bias_operator = sum([FermionOperator(((i, 1), (i, 0)), i) for i
-                             in range(n_qubits)], FermionOperator())
-        bias_sparse = jordan_wigner_sparse(bias_operator)
+        number_sparse = jordan_wigner_sparse(number_operator(n_qubits))
+        bias_sparse = jordan_wigner_sparse(number_operator(n_qubits))
```

Comment:
The coefficient in the original for bias operator is "i" instead of "1", which adds a local potential equivalent to the index rather than being uniform